src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/NewAzureRmVmssConfigCommand.cs

```diff
@@ -399,6 +412,24 @@ private void Run()
                 vSku.Name = this.SkuName;
             }
 
+            if (this.IsParameterBound(c => c.EnableResilientVMCreate))
+            {
+                if (vResiliencyPolicy == null)
+                {
+                    vResiliencyPolicy = new ResiliencyPolicy();
+                }
+                vResiliencyPolicy.ResilientVMCreationPolicy = new ResilientVMCreationPolicy(this.EnableResilientVMCreate.IsPresent);
```

Comment:
we shouldn't use IsPResent even on SwitchParameters because users can pass switchParameters as false as -Parameter:$false. It's rare but it does happen so you should check to see if it is $true.